index.js

```diff
@@ -1,9 +1,10 @@
+import 'babel-polyfill';
 import app from './config/express';
 import config from './config/config';
 import logger from './config/winston';
 
 // make bluebird default Promise
-Promise = require('bluebird'); // eslint-disable-line no-global-assign
+// Promise = require('bluebird'); // eslint-disable-line no-global-assign
```

Comment:
Note I left this commented out but present, but we should actually delete it if we don't want it.
Is there any reason I'm missing that we need to keep bluebird around?

Reply:
Good catch. I see no reason why we need it. The only things we use are `new Promise()` and `Promise.all()` which are implemented natively in node 8.15.0 (I just checked) and probably several earlier versions, too.

Also, I ran `yarn mongo:create_missing_users` which uses `Promise.all()` and it worked.